index.html

```diff
@@ -364,6 +365,11 @@ <h2><dfn>disconnect()</dfn> method</h2>
         <a>relevant global object</a>, and also empty <a>context object</a>'s
         <a>observer buffer</a>.</p>
       </section>
+      <section>
+        <h2><dfn>takeRecords()</dfn> method</h2>
+        <p>The <a>takeRecords()</a> method returns a copy of the <a>observer
```

Comment:
I don't think the observer buffer can be directly returned.

Can we phrase this in terms of step 4.2?

Reply:
Done. Looks almost equivalent to me